disasm.ml

```diff
@@ -72,10 +65,7 @@ let disassemble_instrs buf ?(format_pc = no_line_number) (prog : instructions) =
     | Read var                        -> pr buf " read %s" var
     | Osr {cond; target = {func; version; label}; map} ->
       let dump_var buf = function
-        | Osr_const (x, e)     -> pr buf "const %s = %a" x dump_expr e
-        | Osr_mut (x, e)       -> pr buf "mut %s = %a" x dump_expr e
-        | Osr_mut_ref (x, y)   -> pr buf "mut %s = &%s" x y
-        | Osr_mut_undef x      -> pr buf "mut %s" x
+        | Osr_var (x, e)     -> pr buf "var %s = %a" x dump_expr e
```

Comment:
Again, just wondering if the pattern matching can be done in the parameters. But I guess it might be cleaner this way.